module/GeebyDeeby/view/geeby-deeby/index/index.phtml

```diff
@@ -1,39 +1,77 @@
-<? $this->layout()->title = 'Main Menu' ?>
-<h2>Database</h2>
-<p>List Series
-  [<a href="<?=$this->url('series')?>">by name</a>,
-  <a href="<?=$this->url('categories')?>">by category</a>,
-  <a href="<?=$this->url('cities')?>">by city</a>,
-  <a href="<?=$this->url('countries')?>">by country</a>,
-  <a href="<?=$this->url('languages')?>">by language</a>,
-<!--  <a href="<?=$this->url('materials')?>">by material type</a>, -->
-  <a href="<?=$this->url('publishers')?>">by publisher</a>,
-  <a href="<?=$this->url('series', array('extra' => 'Comments'))?>">with comments</a>]
-</p>
-<p>List People
-  [<a href="<?=$this->url('people')?>">by name</a>,
-  <a href="<?=$this->url('people', array('extra' => 'Bios'))?>">with biographical notes</a>]
-</p>
-<p>List Items
-  [<a href="<?=$this->url('items')?>">by name</a>,
-<!--  <a href="<?=$this->url('platforms')?>">by platform</a>, -->
-  <a href="<?=$this->url('tags')?>">by subject/tag</a>,
-  <a href="<?=$this->url('items', array('extra' => 'ByYear'))?>">by year</a>,
-  <a href="<?=$this->url('items', array('extra' => 'FullText'))?>">with full text</a>,
-  <a href="<?=$this->url('items', array('extra' => 'Reviews'))?>">with reviews</a>]
-</p>
-<!--
-<p><a href="<?=$this->url('files')?>">List Files</a></p>
-<p><a href="<?=$this->url('links')?>">List Links</a></p>
--->
+<? $this->layout()->title = 'Main Menu'; ?>
+<div class="row">
+  <div class="col-md-8">
+    <h2>Welcome!</h2>
+    <p>The Edward T. LeBlanc Dime Novel Bibliography project aims to create a comprehensive online database of dime novels, story papers, reprint libraries and related materials. <a href="<? echo $this->url('about')?>">Learn More &rarr;</a></p>
+    <h2>Browse</h2>
+    <div class="row">
+      <div class="col-sm-6">
+        <h3>Series</h3>
+        <ul>
+          <li><a href="<? echo $this->url('series')?>">by name</a></li>
+          <li><a href="<? echo $this->url('categories')?>">by category</a></li>
+          <li><a href="<? echo $this->url('cities')?>">by city</a></li>
+          <li><a href="<? echo $this->url('countries')?>">by country</a></li>
+          <li><a href="<? echo $this->url('languages')?>">by language</a></li>
+          <!--  <a href="<? echo $this->url('materials')?>">by material type</a></li> -->
+          <li><a href="<? echo $this->url('publishers')?>">by publisher</a></li>
+          <li><a href="<? echo $this->url('series', array('extra' => 'Comments'))?>">with comments</a></li>
+        </ul>
+        <h3>Users</h3>
+        <ul>
+          <li><a href="<? echo $this->url('users')?>">List Registered Users</a></li>
+          <li><a href="<? echo $this->url('reviews')?>">Browse Recent Reviews</a></li>
+        </ul>
+      </div>
+      <div class="col-sm-6">
+        <h3>List Items</h3>
```

Comment:
You shortened some headers to remove "List" but left it on others. Should we be more consistent?